src/script/components/app-card.ts

```diff
@@ -479,16 +478,16 @@ export class AppCard extends LitElement {
         <img
           class=${this.imageClasses()}
           src="${this.imageUrl}"
-          alt="${this.title} card header image"
+          alt="${this.cardTitle} card header image"
```

Comment:
for some reason the app-card was having a property collision with this attribute